ui/src/dummy.test.ts

```diff
@@ -3,3 +3,5 @@ describe('dummy test', () => {
     expect(1 + 2).toEqual(3);
   });
 });
+
+export {};
```

Comment:
Is this required?

Reply:
Yes. Otherwise it gives this error: 
```
'dummy.test.ts' cannot be compiled under '--isolatedModules' because it is considered a global script file. Add an import, export, or an empty 'export {}' statement to make it a module.
```

This is just a dummy test and we can remove it once we start adding some real tests.